src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -77,11 +77,28 @@ class Test(avocadoTest):
     def __init__(self, *args, **kwargs):
         """Initialize a Test object."""
         super(Test, self).__init__(*args, **kwargs)
+
+        # if self.timeout is a string any time value may be entered
+        # but the time speciified must be in this order DHMS
+        # examples   10D2H10M10S or 25M or 2H5S
+        if isinstance(self.timeout, str):
+            self.timeout = self.timeout.lower()
```

Comment:
(pylint-no-member) Instance of 'int' has no 'lower' member

Reply:
self.timeout = str(self.timeout).lower()